time stamp

# number of registered robots
uint32 registered_robots

# States of the current requests
RequestStatus[] requests

# states of the current tasks
TaskState[] tasks

